padertorch/summary/model_info.py

```diff
@@ -0,0 +1,45 @@
+from collections import namedtuple
+from typing import Tuple
+import numpy as np
+from torch import nn
+
+
+def num_parameters(module: nn.Module) -> Tuple[int, int]:
```

Comment:
Idea: Add an argument `unit` with the values `unit in ['count', 'size']`? (You may know better names)